examples/dato/README.md

```diff
@@ -0,0 +1,139 @@
+# Makeswift + Dato Integration
+
+This integration allows you to combine the power of Dato's content management with Makeswift's visual page building capabilities. Visually create dynamic, content-rich websites with a seamless development experience.
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/) 18.x or later
+- [Makeswift](https://www.makeswift.com/) account (for visual page building)
+- [Dato](https://www.datocms.com/) account (for content management)
+- Basic familiarity with [Next.js App Router](https://nextjs.org/docs/app)
+
+This guide assumes that you're familiar with Makeswift and setting up a custom host. Read here to learn more: https://docs.makeswift.com/developer/app-router/installation
+
+## Project Structure
+
+```
+dato/
+├── app/                   # Next.js App Router pages
+│   ├── blog/              # Blog-related routes
+│   ├── api/               # API endpoints
+│   └── [[...path]]/       # Dynamic Makeswift routes
+├── components/            # Reusable React components
+│   └── dato/        # Dato-specific components
+│       ├── common/        # Common scalar components
+│       ├── entries/       # Entry-specific components
+│       │   └── BlogPost/  # BlogPost entry components
+│       └── queries/       # GraphQL queries
+├── lib/                   # Utility functions and configs
+└── generated/             # Auto-generated types from Dato
+```
+
+## Quick Start
+
+### 1. Set up the project
+
+```bash
+npx makeswift@latest init --example=dato
```

Comment:
Similar to above, but I think we should clarify the user to create a new site during this process. Do we want them to use a template? Start from scratch? We should clarify that. Additionally, the API key should be taken care of autmatically in this step right? If so, we should call it out here to let them know it happens and remove the next step